### DISCUSSION TITLE
negative image prompts being ignored

### DISCUSSION THREAD

**Discussion by allroundgoodegg** [lllyasviel/Fooocus#928] [General]
I want a viking woman but with no horns on her helmet (or head where she gets horns anyway, despite not wearing a helmet... that happens a lot too). I put Horns and Helmet into the negative prompt box but keep getting results with... horns. What am I doing wrong?

**Comment by rubenmejiac** [comment]
If you are using the Extreme Speed rendering setting, currently negative prompts are ignored for that mode. 
You can use 'Speed' or 'Quality' for the system to take negatives into consideration. 
If you are already using those, you can try to emphasize what you don't want to see in your negative prompt with ( ), in your example (horns) (helmet)

**Comment by frankyifei** [comment]
try this,  choose speed mode, and in developer debug mode, set sampler and scheduler to lcm, set forced overwrite of sampling steps to 8-12. then set CFG to 1.5. Now your negative prompt should work.

**Reply by JD-Alx** [reply]
I have the same problem, in my case it ignores the negative prompt "Asian face". Your settings didn't help. CFG you mean "cfg mimicking"? And what does lcm change and why do sampler and scheduler use the values that are currently set? Will it not affect the quality of generation? Why exactly are they set there? And how to reset all these values at debug tools to default? Plz halp X_X

**Reply by mashb1t** [reply]
please find a response in your issue https://github.com/lllyasviel/Fooocus/issues/2263
